static/data/profile.json

```diff
@@ -0,0 +1,196 @@
+				"Designed architecture using Microservices Pattern & Domain-Driven Design on Sales Promotion feature for Opn.Store",
+				"Helped, and collaborated with software engineering team on coding with good practices and lowering technical debt",
+				"Researched & introduced modern web technologies to the team, such as SvelteJS, TailwindCSS, Vite, etc.",
+				"Designed, bootstrapped, and developed 4 E-commerce related services for Opn.Store"
+			]
+		},
+		{
+			"position": "Maker, Co-founder",
+			"company": "Maker Zoo",
+			"url": "https://fb.me/makerzoo",
+			"years": ["2014", "2016"],
+			"details": [
+				"Pioneered Maker Movement in Thailand by opening first of the Makerspaces in Bangkok & Thailand",
+				"Hosted maker workshops eg. 3D-modeling, 3D-printing, Arduino, IoT, etc. Having more than 100 participants combined.",
+				"Developed hardware-related products for 3-5 clients and startups",
+				"Developed smart door lock system for Hubba, first co-working space in Bangkok",
+				"Provided 3D-printing service & consultant to more than 50 clients",
+				"Hosted 10+ chapters of tutorial videos on Intel Galileo development board, for Intel Thailand"
+			]
+		},
+		{
+			"position": "Web / IoT Developer (Contract)",
+			"company": "Box24",
+			"url": "https://washbox24.com",
+			"years": ["2015", "2016"],
+			"details": [
+				"Developed internet connected Smart Lockers System using web technologies (Node.js, React.js) on embedded systems (Ubuntu Linux, Arduino, Raspberry Pi)",
+				"Developed private Node.js library to connect with Nayax via serial port to receive cashless payment on vending machines, and also supported Thai Baht cash payment"
+			]
+		},
+		{
+			"position": "Web Developer",
+			"company": "Figabyte",
+			"url": "https://www.facebook.com/figabyte",
+			"years": ["2011", "2014"],
+			"details": [
+				"Developed more than 20 Facebook web apps with CodeIgniter, using Facebook API",
+				"Developed web platform using Ruby on Rails 3 & 4",
+				"Made iOS apps for several clients using PhoneGap and RubyMotion"
+			]
+		},
+		{
+			"position": "Part-time Web Developer",
+			"company": "Wongnai Media (wongnai.com)",
+			"url": "https://wongnai.com",
+			"years": ["2010"],
+			"details": [
+				"Developed a small part of website using Java with Spring MVC Framework built with Apache Maven",
+				"Learned TDD practice & making early-stage startup"
+			]
+		}
+	],
+	"educations": [
+		{
+			"head": "B.Eng. Computer Engineering",
+			"details": "Chulalongkorn University, TH. 2007-2010 (GPA 3.45)"
+		}
+	],
+	"interests": [
+		"Areas : Coding, Productivity & Life-hacks, Mechanical Keyboards",
+		"Typing : 120+ words per minute (Colemak Mod-DH layout)",
+		"Games : Music games, FPS, MOBA, Puzzle",
+		"Music : EDM, Rock, Japanese",
+		"Remote working is a OK. I extensively use self time tracking app for all my works.",
+		"Pets : I have 3 dogs"
+	],
+	"resumeUrl": {
+		"sourceLink": "https://resume.narze.live",
```

Comment:
This line should link to github 
```suggestion
		"sourceLink": "https://github.com/narze/resume",
```

Reply:
<img width="330" alt="image" src="https://github.com/narze/resume/assets/248741/9057c1ba-504b-49b7-bcda-56895f46b255">
the [Data] link should point to https://github.com/narze/resume/blob/main/static/data/profile.json (it'll work after merging to main)